src/test/java/core/basesyntax/suppliers/FigureSupplier.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.enam.FigureList;
+import core.basesyntax.figure.Figure;
+import core.basesyntax.figure.figures.*;
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        switch (FigureList.values()[random.nextInt(FigureList.values().length)].toString()) {
```

Comment:
let's make the choice of a figure based on the random index from 0 to 4, in default let's return default figure